.eslintrc

```diff
@@ -40,15 +40,16 @@
         "operator-linebreak": 0,
         "prefer-template": 0,
         "react/destructuring-assignment": 0,
-        "react/jsx-boolean-value": 0,
         "react/forbid-prop-types": 0,
+        "react-hooks/exhaustive-deps": 1,
```

Comment:
This is just a rearrangement plus the addition of `"react/no-danger": 0` (I love danger)